manifests/params.pp

```diff
@@ -21,20 +21,25 @@
   $use_scl_ruby = true
   $scl_ruby = 'rh-ruby25'
 
-  $scl_nodejs = 'rh-nodejs12'
-  $scl_postgresql = 'rh-postgresql12'
+  if $facts['os']['release']['major'] == '7' {
+    $scl_nodejs = 'rh-nodejs12-'
+    $scl_postgresql = 'rh-postgresql12-'
```

Comment:
This is a bit inconsistent with the above `$scl_ruby` which doesn't use the `-` suffix which is a bit annoying.

Reply:
A bit, but these are are also not inputs from the user (which none of them should be).

Reply:
Still, I strongly dislike inconsistency. This also means you can't enable it by default. For example, in forklift we pass the Ruby version here:
https://github.com/theforeman/forklift/blob/b85b14b7f894b9ee1193fa15f8780d887f13388c/roles/katello_devel/meta/main.yml#L18

It can't have the dash since the variable is also used here:
https://github.com/theforeman/forklift/blob/b85b14b7f894b9ee1193fa15f8780d887f13388c/roles/ruby_scl/tasks/main.yml#L14-L22

Now we do the same thing here:
https://github.com/theforeman/forklift/blob/b85b14b7f894b9ee1193fa15f8780d887f13388c/roles/nodejs_scl/tasks/main.yml#L12-L18

That's why I think they must be parameters. That way they remain in sync.

The alternative is to kill the nodejs_scl and ruby_scl Ansible roles and do it all in this module.